pt-BR/mail/chrome/messenger/messenger.dtd

```diff
@@ -666,15 +666,15 @@
 <!ENTITY SendMailTo.accesskey "N">
 <!ENTITY CopyEmailAddress.label "Copiar email">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY CopyEmailAddress.label "Copiar endereço de correio eletrônico">